### PR TITLE
Tic tac toe

### PR DESCRIPTION
**Pull request by SeanPhilippi**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 03week/ticTacToe.js [hunk 1]
```diff
@@ -23,33 +32,79 @@ function printBoard() {
...
+  if (playerTurn === 'X') {
+    playerTurn = 'O';
+  } else {
+    playerTurn = 'X';
+  }
+}
+
+const horizontalWin = () => {
+  // check that index in array has a mark in it
+  const win1 = board[0][0] !== ' ' && board[0][0] === board[0][1] && board[0][0] === board[0][2];
```
Comment (reneemeyer) [inline 03week/ticTacToe.js]: nice, how could you simply more?

**Review by reneemeyer — APPROVED** [review]
Nice job! Merge it

**Comment by SeanPhilippi** [comment]
const turnSwitcher = () => {
  return playerTurn === 'X' ? playerTurn = 'O' : playerTurn = 'X';
}

how's that?

**Comment by SeanPhilippi** [comment]
Actually, this:

const turnSwitcher = () => playerTurn === 'X' ? playerTurn = 'O' :
playerTurn = 'X';

**Pull request merged**